viewer/packages/utilities/src/three/ThickLine.ts

```diff
@@ -5,38 +5,43 @@
 import * as THREE from 'three';
 import { LineGeometry, LineMaterial, Line2 } from 'three-stdlib';
 
-export class MeasurementLine {
+/**
+ * A class representing 3D objects constituting a line supporting widths larger than 1
+ */
+export class ThickLine {
```

Comment:
Do you think that the name sounds a little strange? In theory `ThickLine` can be not very thick (width = 1), also "thick" is quite vague term. What do you think about renaming it to something like `DynamicWidthLine`, `NonStaticWidthLine` or `AdaptiveWidthLine`/`AdaptiveLine`?

Reply:
I agree, I'll change it up. Also, will refactor the entire class